src/foam/java/refinements.js

```diff
@@ -675,6 +675,56 @@ foam.CLASS({
   ],
 
   methods: [
+    function buildMethodInfoInitializer(cls) {
+      // Add MethodInfo field for each method
+      initializerString = `new foam.core.MethodInfo(){
+        @Override
+        public String getName(){
+          return \"${this.name}\";
```

Comment:
Are the \'s required?

Reply:
Are the backslashes required?

Reply:
it's a string so it should be surrounded by double quotes. And the backslashes are there to escape the "

Reply:
But it isn't inside a " quoted string, it's inside a ` quoted string, so the " has no special meaning and doesn't need to be escaped. As far as I know.

Reply:
ok